packages/component-library/src/FooterNew/FooterNew.js

```diff
@@ -98,6 +98,9 @@ const Footer = props => (
       </div>
       <Logo css={logoStyle(props)} type="squareLogo" />
     </div>
+    <a href="https://www.netlify.com">
+      <img src="https://www.netlify.com/img/global/badges/netlify-color-bg.svg"/>
+    </a>                 
```

Comment:
Is this required with the nonprofit package?

Reply:
There's a requirement to have a link for the open-source plan (https://www.netlify.com/legal/open-source-policy/). I'm going to tone it down a bit, though

Reply:
Actually, removed this for now.